src/ingest_validation_tools/table-schemas/includes/fields/resolution_z_unit.yaml

```diff
@@ -0,0 +1,30 @@
+# af-v0.yaml; af-v1.yaml; seqfish-v0.yaml; seqfish-v1.yaml
+- name: resolution_z_unit
+  description: The unit of incremental distance between image slices.(um)
+  constraints:
+    enum:
+      - nm
+      - um
+
+# stained-v0.yaml; stained-v1.yaml
+- name: resolution_z_unit
+  description: The unit of incremental distance between image slices.
+  constraints:
+    enum:
+      - nm
+      - um
+
+# lightsheet-v0.yaml; lightsheet-v1.yaml
+- name: resolution_z_unit
+  description: The unit of z resolution.
```

Comment:
resolution_z_distance has a different meaning for light sheet than for other imaging assays because it references optical resolution rather than a physical distance between image slices. resolution_z_distance & unit should be required for light sheet.